src/metis/cli/utils.py

```diff
@@ -106,6 +106,20 @@ def pretty_print_reviews(results, quiet=False):
                         f"    [cyan]Line number:[/cyan] {r['line_number']}",
                         quiet,
                     )
+                if r.get("cwe"):
+                    print_console(f"    [red]CWE:[/red] {r['cwe']}", quiet)
+                severity = r.get("severity")
+                if severity:
```

Comment:
```suggestion
                if severity := r.get("severity"):
```

You could use the walrus operator here. Some people might not find it readable, it was introduced in 3.8 so maybe it's not so bad.